Groupon UP/Groupon UP/UpInvitation.swift

```diff
@@ -15,7 +15,7 @@ class UpInvitation {
         return object.objectId!
     }
     var message: String! {
-        return object["message"].stringValue
+        return object["message"] as! String
```

Comment:
SwiftJSON's `.stringValue` is safer than as! String, since it returns "" for none existing case without crashing the app